src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -245,17 +240,19 @@ export class CalciteTipManager {
       return <Host />;
     }
     return (
-      <Host onKeydown={this.tipManagerKeyDownHandler} tabindex={this.tabindex}>
-        <header class={CSS.header}>
-          <h2 class={CSS.heading}>{this.groupTitle}</h2>
-          <calcite-action text={this.textClose} onClick={this.hideTipManager} class={CSS.close}>
-            <CalciteIcon size="16" path={x16} />
-          </calcite-action>
-        </header>
-        <div class={classnames(CSS.tipContainer, this.direction)} key={this.selectedIndex}>
-          <slot />
+      <Host onKeydown={this.tipManagerKeyDownHandler}>
```

Comment:
Minor: you could move the `onKeyDown` to the element that receives focus but it will bubble up in this case.